### ISSUE TITLE
-Wuninitialized in sound/soc/codecs/aw88399.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#1952]
```
sound/soc/codecs/aw88399.c:441:18: error: variable 'vsense_select' is uninitialized when used here [-Werror,-Wuninitialized]
  441 |         vsense_select = vsense_select & (~AW88399_VDSEL_MASK);
      |                         ^~~~~~~~~~~~~
sound/soc/codecs/aw88399.c:431:28: note: initialize the variable 'vsense_select' to silence this warning
  431 |         unsigned int vsense_select, vsense_value;
      |                                   ^
      |                                    = 0
1 error generated.
```

**Comment by nathanchance** [comment]
Patch submitted: https://lore.kernel.org/20231027-asoc-aw88399-fix-wuninitialized-v1-1-b1044493e4cd@kernel.org/

**Comment by nathanchance** [comment]
https://git.kernel.org/next/linux-next/c/cba4590036855f4e3110d43c14385d2401080dbb
